customization/includes.chroot/etc/jaiabot/init/first-boot.preseed.ex

```diff
@@ -0,0 +1,33 @@
+# Preseed configuration. Booleans can be "true" or "yes"
+# This is a bash script so any bash command is allowed
+# Edit and rename to "/etc/jaiabot/init/first-boot.preseed" to take effect
```

Comment:
Per our conversion we should move this to /boot/jaiabot/init

Reply:
This is completed.